src/es/leercapitulo/src/eu/kanade/tachiyomi/extension/es/leercapitulo/LeerCapitulo.kt

```diff
@@ -177,17 +179,24 @@ class LeerCapitulo : ParsedHttpSource() {
 
         val arrayData = document.selectFirst("#array_data")!!.text()
 
-        val scriptUrl = document.selectFirst("section.bodycontainer > script[src$=.js]")?.attr("abs:src")
-            ?: throw Exception("Script not found")
+        val scripts = document.select("head > script[src^=/assets/][src$=.js]")
+
+        var dataScript: String? = null
 
-        val scriptData = client.newCall(GET(scriptUrl, headers)).execute().body.string()
+        for (script in scripts) {
+            val scriptData = notRateLimitClient.newCall(GET(script.attr("abs:src"), headers)).execute().body.string()
```

Comment:
Would it make sense to cache this for a short while, like the index of the correct script? Not as in filtering out, but rather changing the order scripts are requested.